docs/components/FileUploadDocs.js

```diff
@@ -26,21 +26,11 @@ class FileUploadDocs extends React.Component {
         <h3>Demo</h3>
         <div style={{ cursor: 'pointer', textAlign: 'center', width: '80%', margin: 'auto' }}>
           <FileUpload
-            allowedFileTypes={['image/jpeg', 'text/csv', 'image/png']}
-            imageValidation={{
-              exactHeight: 200,
-              exactWidth: 400,
-              maxHeight: 400,
-              maxWidth: 800,
-              minHeight: 100,
-              minWidth: 200,
-              ratioHeight: 1,
-              ratioWidth: 2
-            }}
+            allowedFileTypes={['image/jpeg', 'text/csv', 'image/png', 'gif']}
```

Comment:
Actually, this was my lazy attempt to validate an extension instead of a filetype. So it would check `something.gif` as valid.